.github/workflows/doc-tests.yml

```diff
@@ -26,22 +23,31 @@ jobs:
       - uses: actions/setup-python@v6
         with:
           python-version: 3.x
-          cache: 'pip'
 
-      - name: Install hatch
-        run: pip install hatch==1.15.1
+      - uses: astral-sh/setup-uv@v7
+        with:
+          activate-environment: true
+
+      - name: Install tmt[docs]
+        run: uv sync --frozen --extra docs
 
       - name: Run sphinx html builder
-        run: hatch run docs:html -W
+        run: sphinx-build -b html docs/ docs/_build -W
 
       - name: Cache linkcheck results
         uses: actions/cache@v4
         with:
           path: docs/_build/linkcheck_cache.json
           key: linkcheck
 
-      - name: Run sphinx linkcheck
-        run: hatch run docs:linkcheck
+      - name: Run sphinx html builder
```

Comment:
The step name says `html builder` but the command runs linkcheck. Rename it to Run sphinx linkcheck to avoid confusion in CI logs.

Reply:
fixed 3464f9b